pkg/options/options.go

```diff
@@ -77,6 +79,7 @@ func FromEnv(init bool) (*Options, error) {
 	retOptions.UseSpotInstance = os.Getenv(AWS_USE_SPOT_INSTANCE) == "true"
 	retOptions.UseSessionManager = os.Getenv(AWS_USE_SESSION_MANAGER) == "true"
 	retOptions.KmsKeyARNForSessionManager = os.Getenv(AWS_KMS_KEY_ARN_FOR_SESSION_MANAGER)
+	retOptions.Route53ZoneName = os.Getenv(AWS_ROUTE53_ZONE_NAME)
```

Comment:
Out of scope for this PR but in the future we should add some validation at this stage in case people have configured multiple ways of connecting to workspaces 
(public Ip, private ip, instance connect, session manager, route53)

Reply:
agree